Week 1/Weekly Practice Problems (Week 1)/2011: Final Value of Variable After Performing Operations/Mohammad.cpp

```diff
@@ -0,0 +1,23 @@
+class Solution {
+public:
+    int finalValueAfterOperations(vector<string>& operations) {
+        
+     int x = 0; 
+     for(string i : operations){
+         if(i == "++X" || i == "X++"){
+             x++;
+         }
+         if(i == "--X" || i == "X--"){
+              x--;
+         }
+       
+            
+         }
```

Comment:
This is a simple program, so it's pretty easy to tell where the braces are, but in longer chunks of code it would be good to have a bit more readable indenting and spacing:
```suggestion
     for(string i : operations){
         if(i == "++X" || i == "X++"){
             x++;
         }
         if(i == "--X" || i == "X--"){
             x--;
         }
     }
```

Reply:
Agreed. Maybe add an else on L10 too? :)